### PR TITLE
Add SBMTD sample data

### LINKED ISSUE #363: Add SBMTD sample data to CSV
for https://github.com/cal-itp/benefits/issues/1664

the data needs to have 4-digit number IDs, not 5 (which is what MST has).

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #363

### CODE REVIEW

File: data/server.csv [hunk 1]
```diff
@@ -15,6 +15,12 @@ sub,name,type
 71650,De la Cruz,courtesy_card
 03218,D'Andrea,courtesy_card
 51828,Grafton-Herbert,courtesy_card
+1234,Barbara,mobility_pass
+2345,Brown,mobility_pass
+3456,Calderón,mobility_pass
+4567,Van Buren,mobility_pass
+4178,Núñez,courtesy_card
+0328,D'Andrea,courtesy_card
 00596884ca82d6df736892667163ea13ca95404d3f746e516d2443a0c12cc9e199a8bf113142ebe0bbdbbdcecde179fd1ba45c226033d88cec5606b1e29517d6,b654c445c17985721f9899fb310a320fb9f55a1fde191327aaf241aeabd177172e5cef8bf3b8d628213c62d84d4aff8f50e42d5bc849784a079fe84d8fad0de0,courtesy_card
```
Comment (machikoyasuda) [inline data/server.csv:24]: @thekaveman @angela-tran is this okay for now? I just made it up. I can add hashed values later. Just want to get this up for easier testing.

File: data/server.csv [hunk 2]
```diff
@@ -15,6 +15,12 @@ sub,name,type
 71650,De la Cruz,courtesy_card
 03218,D'Andrea,courtesy_card
 51828,Grafton-Herbert,courtesy_card
+1234,Barbara,mobility_pass
+2345,Brown,mobility_pass
+3456,Calderón,mobility_pass
+4567,Van Buren,mobility_pass
+4178,Núñez,courtesy_card
+0328,D'Andrea,courtesy_card
```
Comment (thekaveman) [inline data/server.csv:23]: Typo in these last two? `courtesy_card`??

Comment (machikoyasuda) [reply data/server.csv:23]: oh oops!!

Comment (machikoyasuda) [reply data/server.csv]: fixed https://github.com/cal-itp/eligibility-server/pull/365/commits/4144a8d22196c0359095d1fdc653a317820c408b

**Review by angela-tran — APPROVED** [review]

**Pull request merged**